core/backend/src/test/integration/IModelTransformerHub.test.ts

```diff
@@ -42,10 +42,22 @@ describe("IModelTransformerHub (#integration)", () => {
     }
   });
 
-  it("Transform source iModel to target iModel", async () => {
+  // Fails with,
+  //    IModelTransformerHub (#integration)
+  //         Transform source iModel to target iModel:
+  //       Not Found: CodeSpec not found
+  //        at /Users/calebshafer/dev/github/imodeljs/core/backend/src/CodeSpecs.ts:36:15
+  //        at BriefcaseDb.withPreparedStatement (src/IModelDb.ts:262:22)
+  //        at CodeSpecs.queryId (src/CodeSpecs.ts:33:25)
+  //        at CodeSpecs.getByName (src/CodeSpecs.ts:80:29)
+  //        at IModelCloneContext.remapCodeSpec (src/IModelCloneContext.ts:54:62)
+  //        at TestIModelTransformer.initCodeSpecRemapping (src/test/IModelTransformerUtils.ts:1329:18)
+  //        at new TestIModelTransformer (src/test/IModelTransformerUtils.ts:1311:10)
+  //        at Context.<anonymous> (src/test/integration/IModelTransformerHub.test.ts:117:29)
+  it.skip("Transform source iModel to target iModel", async () => {

```

Comment:
@scsewall FYI, I turned this one off for now. I didn't get a chance to look into it much.

Reply:
That test has been running consistently for a long time (not flaky) and tests real iModelHub workflows. Any failure is likely a real problem. I can take a look if you want.

Reply:
Ah okay, it seems to have popped up in the past few weeks.  I don't remember seeing it a month ago when I started this project of cleaning up the tests.

If you could take a look, that'd be great. Thanks!

Reply:
I have essentially the same test that runs 2 ways:

1. With snapshot iModels
2. With briefcases + iModelHub integration

Case #1 is passing.
Case #2 is not which suggests a problem with our briefcase or iModelHub integration.

Anyway, this seems like a real problem, but I reproduced it without any of your changes. I will keep digging...